screens/MainComponent.js

```diff
@@ -49,6 +51,32 @@ const DirectoryNavigator = () => {
     );
 };
 
+const AboutNavigator = () => {
+    const Stack = createStackNavigator();
+    return (
+        <Stack.Navigator screenOptions={screenOptions}>
+            <Stack.Screen
+                name='About'
+                component={AboutScreen}
+            // options={{ title: 'About' }}
```

Comment:
It is good practice to remove commented out bits of code.  Generally if you need to revert back to a piece of code, you can find it in previous commits.

Reply:
Gotcha!